CHANGELOG

```diff
@@ -2,6 +2,17 @@ libseccomp: Releases
 ===============================================================================
 https://github.com/seccomp/libseccomp
 
+* Version 2.5.2 - August 27, 2021
+- Update the syscall table for Linux v5.14-rc7
+- Add a function to the Python bindings to get the nofication file descriptor
```

Comment:
Is it worth explicitly mentioning the function name (`get_notify_fd()`)?

Reply:
Good idea.  I'll add it.  Thanks!